CONTRIBUTING.md

```diff
@@ -27,49 +27,36 @@ Random order of test execution is provided by pytest-randomly, so more details a
 
 ## Integration tests
 
-Every integration test should have `integration` mark. By default, integration tests are not execute when running `pytest`.
+Integration tests are running against real Snowflake instance. Before running them you need to configure the connection to Snowflake.
 
-To execute only integration tests run `pytest -m integration`.
+To run integration tests run `pytest -m integration`.
 
-### Connection parameters in `config.toml`
+Integration tests live in `tests_integration` directory. Every integration test should have `integration` mark.
+By default, integration tests are not execute when running `pytest` command.
 
-Add the following connection to your `config.toml`
+### Configuring integration tests connection
+For running integration tests you will need to prepare your account. To do so you can execute the sql script
+located in `test_integration/scripts/integration_account_setup.sql`. It will create required users, roles,
```

Comment:
Should we add here exception with `spcs service` which requires docker image in repository?